seg_utils/ui/label_ui.py

```diff
@@ -100,10 +101,34 @@ def setupUI(self, mainWindow):
         self.polyLabel.setObjectName("polyLabel")
         self.polyLabel.setText("Polygons")
         self.polyLayout.addWidget(self.polyLabel)
+
+        # polySubFrame comprises both the PolygonList and the corresponding "Add comment"-List
```

Comment:
should probably make this it's own class to help clean up the main layout setup and since the data deals with each other.

Reply:
an own class for the entire PolyFrame (including polyList, commentList and header line) or just the subFrame (polyList and commentList?

Reply:
@jakobhampel I would include it all because you'll never use the header without the other stuff. And if you use this widget again somewhere, the header can always be suppressed if desired.